x-pack/plugins/timelines/server/search_strategy/timeline/factory/types.ts

```diff
@@ -5,17 +5,22 @@
  * 2.0.
  */
 
-import { IEsSearchResponse } from '../../../../../../../src/plugins/data/common';
+import { JsonObject } from '@kbn/common-utils';
+import {
+  IEsSearchResponse,
+  ISearchRequestParams,
+} from '../../../../../../../src/plugins/data/common';
 import {
   TimelineFactoryQueryTypes,
   TimelineStrategyRequestType,
   TimelineStrategyResponseType,
 } from '../../../../common/search_strategy/timeline';
 
 export interface TimelineFactory<T extends TimelineFactoryQueryTypes> {
-  buildDsl: (options: TimelineStrategyRequestType<T>) => unknown;
+  buildDsl: (options: TimelineStrategyRequestType<T>) => ISearchRequestParams;
   parse: (
     options: TimelineStrategyRequestType<T>,
-    response: IEsSearchResponse
+    response: IEsSearchResponse,
+    authFilter?: JsonObject | undefined
```

Comment:
nit -> if it is already a conditional parameter,  do we really need the undefined?

Reply:
Gah! Actually meant to remove that. We do not want to include the authFilter in the inspect. At least in my opinion it seems odd that we would include it in the inspect, but let me know if I'm wrong. It's not in there right now. I was going to add it and then didn't and forgot to remove the typing here.